ios/FluentUI/People Picker/ContactView.swift

```diff
@@ -9,22 +9,18 @@ import UIKit
 
 public extension Colors {
     struct Contact {
-        public static var title: UIColor = textPrimary
-        public static var subtitle: UIColor = textSecondary
+        public static let title: UIColor = textPrimary
+        public static let subtitle: UIColor = textSecondary
+        public static let pressedState = UIColor(red: 0.83, green: 0.83, blue: 0.83, alpha: 0.6)
+        public static let unpressedState = UIColor(red: 1.0, green: 1.0, blue: 1.0, alpha: 0.0)
```

Comment:
white with alpha?

Reply:
I'm not too familiar with colours, so I want to confirm that you want me to use `UIColor(white: CGFloat, alpha: CGFloat)` instead of what I currently have?

Reply:
but then again do we need this color? if you aren't pressing, isn't it just clear color?

Reply:
This colour is to set the view back to invisible when the contact view isn't being tapped. I think we need it since if we don't set it back, the avatar image will still have a grey overlay when it isn't being tapped.